index.html

```diff
@@ -70,5 +70,220 @@ <h3>LET'S CONNECT</h3>
+				<img src="assets/img/Project4.svg" alt="project four" />
+				<h2>Multi-Post Stories</h2>
+				<ul class="works__card--info">
+					<li><span>CANOPY</span></li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>Back End Dev</p>
+					</li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>2015</p>
+					</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul class="works__card--tags">
+					<li>HTML</li>
+					<li>CSS</li>
+					<li>JavaScript</li>
+				</ul>
+				<button type="button">See project</button>
+			</div>
+		</section>
+
+		<section class="about" id="about">
+			<div class="about__main-content">
+				<h2>About Myself</h2>
+				<p>
+					I’m a software developer! I can help you build a product , feature or
+					website Look through some of my work and experience! If you like what
+					you see and have a project you need coded, don’t hesitate to contact
+					me.
+				</p>
+				<h3>LET'S CONNECT</h3>
+				<ul class="about__main-content--social">
+					<li>
+						<a href="#">
+							<img src="assets/img/Github.svg" alt="github" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/Linkedin.svg" alt="linkedin" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/AngelList.svg" alt="angel list" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/Twitter.svg" alt="twitter" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/Medium.svg" alt="medium" />
+						</a>
+					</li>
+				</ul>
+				<button type="button">Get my resume</button>
+				<ul class="about__main-content--skills">
+					<li>
+						<label for="languages">Languages</label>
+						<input type="checkbox" id="languages" class="menu-item-checkbox" />
+						<img
+							class="menu-item-icon"
+							src="assets/img/Disabled.svg"
+							alt="arrow"
+						/>
+						<ul class="about__main-content--skills--list">
+							<li>
+								<img src="assets/img/JS.svg" alt="javascript" />
+								JavaScript
+							</li>
+							<li>
+								<img src="assets/img/Html.svg" alt="html" />
+								HTML
+							</li>
+							<li>
+								<img src="assets/img/Css.svg" alt="css" />
+								CSS
+							</li>
+						</ul>
+					</li>
+					<li>
+						<label for="frameworks">Frameworks</label>
+						<input type="checkbox" id="frameworks" class="menu-item-checkbox" />
+						<img
+							class="menu-item-icon"
+							src="assets/img/Disabled.svg"
+							alt="arrow"
+						/>
+						<ul class="about__main-content--skills--list">
+							<li>
+								<img src="assets/img/js.svg" alt="react" />
+								React
+							</li>
+							<li>
+								<img src="assets/img/js.svg" alt="vue" />
+								Vue
+							</li>
+							<li>
+								<img src="assets/img/js.svg" alt="angular" />
+								Angular
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you use the correct name for your image path, as you may notice how they are broken. was supposed to be `JS` and not `js`
![image](https://user-images.githubusercontent.com/97100091/227612541-14130724-ca47-4043-87de-51c718d6ec7c.png)